kube-aws-updater

```diff
@@ -129,77 +130,118 @@ label_for_cycling() {
+    jq -r '.Reservations[].Instances[].InstanceId')
   echo "terminating ${node} with instance-id ${instance_id}"
   aws ${aws_opts} ec2 terminate-instances --instance-ids=${instance_id}
-  if [[ ${failDrain} == true ]]; then
-    echo "error on drain, terminating and sleeping ${timeout} seconds"
-    sleep ${timeout}
-  fi
 }
 
 cycle_nodes() {
   local role=$1
 
   # - double ASG size
   local instance_address=$(kubectl --context=${kube_context} get nodes -l role=${role},retiring=${retire_time} -o json | jq -r '.items[0].metadata.name')
-  local instance_id=$(aws ${aws_opts} --output=json ec2 describe-instances --filters "Name=network-interface.private-dns-name,Values=${instance_address}" \
-    | jq -r '.Reservations[].Instances[].InstanceId')
-  local asg_name=$(aws ${aws_opts} --output=json autoscaling describe-auto-scaling-groups \
-    | jq -r '."AutoScalingGroups"[] | select(."Instances"[]."InstanceId"=="'${instance_id}'") | ."AutoScalingGroupName"')
-  local asg_count=$(aws ${aws_opts} --output=json autoscaling describe-auto-scaling-groups \
-    | jq -r '."AutoScalingGroups"[] | select(."Instances"[]."InstanceId"=="'${instance_id}'") | ."DesiredCapacity"')
-  aws ${aws_opts} --output=json autoscaling update-auto-scaling-group --auto-scaling-group-name "${asg_name}" --desired-capacity $(( ${asg_count} * 2)) --max-size $(( ${asg_count} * 2))
+  local instance_id=$(aws ${aws_opts} --output=json ec2 describe-instances --filters "Name=network-interface.private-dns-name,Values=${instance_address}" |\
+    jq -r '.Reservations[].Instances[].InstanceId')
+  local asg_name=$(aws ${aws_opts} --output=json autoscaling describe-auto-scaling-groups |\
+    jq -r '."AutoScalingGroups"[] | select(."Instances"[]."InstanceId"=="'${instance_id}'") | ."AutoScalingGroupName"')
+  local asg_count=$(aws ${aws_opts} --output=json autoscaling describe-auto-scaling-groups |\
+    jq -r '."AutoScalingGroups"[] | select(."Instances"[]."InstanceId"=="'${instance_id}'") | ."DesiredCapacity"')
+
+  aws ${aws_opts} --output=json autoscaling update-auto-scaling-group \
+    --auto-scaling-group-name "${asg_name}" \
+    --desired-capacity $(( ${asg_count} * 2)) \
+    --max-size $(( ${asg_count} * 2))
 
   # - wait for 2x group
   local count=0
   while [[ ${count} -lt ${asg_count} ]]; do
     sleep 8
-    count=$(kubectl --context=${kube_context} get node -lrole=${role} -Lrole,retiring \
-      | grep -v ${retire_time} \
-      | grep -v NotReady \
-      | tail -n +2 \
-      | wc -l)
+    count=$(kubectl --context=${kube_context} get node -lrole=${role} -Lrole,retiring |\
+      grep -v ${retire_time} |\
+      grep -v NotReady |\
+      tail -n +2 |\
+      wc -l)
     echo "${count}/${asg_count} ready ${role}s"
   done
 
   # - stop ASG actions
-  aws ${aws_opts} --output=json autoscaling suspend-processes --auto-scaling-group-name "${asg_name}"
+  aws ${aws_opts} --output=json autoscaling suspend-processes \
```

Comment:
Maybe a comment that we actually need to keep `Terminate` enabled here would be handy for people that are not aware how we decide what to disable. As the first thing that comes in mind is to disable all.